ui/apps/dashboard/src/gql/graphql.ts

```diff
@@ -1024,9 +1025,9 @@ export type RunsFilter = {
 };
 
 export type RunsFilterV2 = {
-  appID?: InputMaybe<Scalars['UUID']>;
+  appIDs?: InputMaybe<Array<Scalars['UUID']>>;
   from: Scalars['Time'];
-  functionID?: InputMaybe<Scalars['UUID']>;
+  functionIDs?: InputMaybe<Array<Scalars['UUID']>>;
   query?: InputMaybe<Scalars['String']>;
```

Comment:
Just flagging this unrelated change, I don't think it should cause any issues though